contracts/tonomy/include/tonomy/tonomy.hpp

```diff
@@ -268,6 +292,23 @@ namespace tonomysystem
       // Following line needed to correctly generate ABI. See https://github.com/EOSIO/eosio.cdt/issues/280#issuecomment-439666574
       typedef eosio::multi_index<"resconfig"_n, resource_config> resource_config_table_dump;
 
+      // Define the structure of a staking allocation
```

Comment:
i would also like us to add another table, a singleton that contains the total amount of tokens are currently staked/unstaking which should be updated when the member functions are called.